lib/simp/cli/environment/secondary_dir_env.rb

```diff
@@ -119,7 +119,7 @@ def apply_facls(path, facl_file)
 
       Dir.chdir(path) do
         info("Applying FACL rules to '#{path}'".cyan)
-        cmd = "setfacl --restore=#{facl_file} 2>/dev/null"
+        cmd = "setfacl --restore=#{facl_file}"
         unless execute(cmd)
           fail(Simp::Cli::ProcessingError, "ERROR:  Failed to apply FACL rules to #{path}")
```

Comment:
So, the issue with failing on this is that if someone removes a file that is normally present during the FACL application, you'll end up with an error.

I'm thinking we have one of two choices.

1. We collect the messages and strip out any that are `No such file or directory`, warn on those and fail if any remain.
2. We simply call this a warning in big letters and tell the user what happened.

Reply:
If you have an error (non-zero return code) the execute() will fail.  All this does is emit error messages if an error occurs.

Reply:
@lnemsick-simp Ah...right. Hrm...well that seems bad because of what I mentioned, but I'll approve it because it's not changing the existing behavior and is certainly more helpful (though potentially quite noisy).